molecule/driver/openstackdriver.py

```diff
@@ -253,14 +258,61 @@ def _set_keypair(self):
             self._openstack.create_keypair(kpn, open(pub_key_file,
                                                      'r').read().strip())
 
+    def reset_known_host_key(self, hostname):
+        return os.system('ssh-keygen -R {}'.format(hostname))
+
+    def check_ssh_availability(hostip, user, timeout, sshkey_filename):
+        import socket
```

Comment:
Lets move this import to the top of the file in alphabetical order, like we do with the rest of our imports.